assets/translations/en-us.yaml

```diff
@@ -1235,6 +1235,10 @@ validation:
   required: '"{key}" is required'
   requiredOrOverride: '"{key}" is required or must allow override'
   service:
+    clusterIp:
+      none: 'Service Type is Headless which requires Cluster IP to be set to "None"'
```

Comment:
Do we even offer the choice to set this if it must be none?  When would this error come up?

Reply:
I was thinking that we could possibly hit this error if the service was created incorrectly outside the UI and subsequently edited in the UI. I am not sure the Kube API would even let this happen now that I type this out though, so I'll drop this check.